api/pkg/filtermanager/filtermanager_latest.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	supportGettingHeadersOnLog = false
+	supportGettingHeadersOnLog = true
```

Comment:
Only the unreleased Envoy version supports getting headers OnLog: 
https://github.com/mosn/htnn/blob/8ded9c5688fb93b3905a61504419071b73ba7409/api/pkg/filtermanager/filtermanager_dev.go#L24

Reply:
I would like to ask why supportGettingHeadersOnLog = true in envoydev and false in !envoydev, do I need to leave it as is?

Reply:
Well I guess I should have known, since !envoydev itself provides the ability to cache the headers on the go side without us needing to judge it by !supportGettingHeadersOnLog, I'll leave them as they are and just remove the `&& m.DebugModeEnabled()`.

Reply:
envoydev => unreleased Envoy version, support getting headers in OnLog phase.